### DISCUSSION TITLE
New page not deploying

### DISCUSSION THREAD

**Discussion by LoreleiAurora** [community/community#21908] [Other Feature Feedback, Questions, & Ideas]
<p>I have created a new <code>gh-pages</code> branch <a href="https://github.com/howtoadhd/satis/tree/gh-pages" rel="nofollow noopener">here</a>, it has been over an hour and the site has still not been deployed. Any ideas why this might be?</p>
<p>The deploy settings are as follows </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/dddd99bde909b9f7821954cde89d93bf7698676a.png" data-download-href="/uploads/short-url/vEIitnNiEbSBKJ1b1wRobC18ohs.png?dl=1" title="Screen Shot 2018-01-02 at 12.53.17.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3892265/181067498-aff89fc5-6468-4de5-81f1-2b3d98156601.png" alt="Screen Shot 2018-01-02 at 12.53.17.png" data-base62-sha1="vEIitnNiEbSBKJ1b1wRobC18ohs" width="690" height="306" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/dddd99bde909b9f7821954cde89d93bf7698676a_2_690x306.png, https://aws1.discourse-cdn.com/github/optimized/1X/dddd99bde909b9f7821954cde89d93bf7698676a_2_1035x459.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/dddd99bde909b9f7821954cde89d93bf7698676a_2_1380x612.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/dddd99bde909b9f7821954cde89d93bf7698676a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-01-02 at 12.53.17.png</span><span class="informations">1504×668 53.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Peace Love &amp; Pogo Sticks</p>
<p>Lorelei xXx</p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/loreleiaurora">@loreleiaurora</a> this was actually discussed in another thread already here: <a href="https://github.community/t/create-pages-repository-but-not-published-yet/10255">https://github.community/t5/Pages/Create-pages-repository-but-not-published-yet/m-p/933#M97</a>. I think the solution there might be helpful to you :smiley:</p>

**Comment by birdcar** [comment, marked as answer]
<p>Hi <a class="mention" href="/loreleiaurora">@loreleiaurora</a>! </p>
<p>I just took a peek at your GitHub Pages site and it appears that it’s publishing now.</p>
<p><a href="https://howtoadhd.github.io/satis/" rel="nofollow noopener">https://howtoadhd.github.io/satis/</a></p>
<p>Glad you were able to resolve the issue! </p>
